index.html

```diff
@@ -495,6 +495,265 @@ <h2>
         </p>
       </section>
     </section>
+    <section>
+      <h2>
+        <code><dfn>user_preferences</dfn></code> member
+      </h2>
+      <p>
+        The `user_preferences` member of the [=application manifest=] is an
+        object that can be used to override values of manfiest members depending
+        on which user preferences are set. It has the following members:
```

Comment:
For: 
> "on which user preferences are set"

We should check the language that CSS uses for this, as it's not clear what "user preferences" means.